packages/rich-text/src/helpers/getLinkedContentTypeIdsForNodeType.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable you-dont-need-lodash-underscore/find */
```

Comment:
We are using this function on `HyperlinkModal.tsx` already. We need to remove it from there too.